docs/manual/forms/examples/repeating_groups.rst

```diff
@@ -0,0 +1,93 @@
+.. _examples_repeating_groups:
+
+================================
+Formulier met herhalende groupen
+================================
+
+In dit voorbeeld maken we een fictief formulier bestaande uit 1 stap met
+een herhalende group component. We gaan berekeningen laten zien die gebruike maken van de waarden
+ingevuld in de herhalende group.
+
+In dit voorbeeld gaan we er van uit dat u een
+:ref:`formulier met berekeningen <examples_calculations>` kan maken en dat
+u op de hoogte bent van hoe :ref:`logica <manual_logic>` werkt.
+
+Formulier maken
+===============
+
+#. Maak een formulier aan met de volgende gegevens:
+
+   * **Naam**: Herhalende group demo
+
+#. Klik op het tabblad **Stappen en velden**.
+#. Klik aan de linkerkant op **Stap toevoegen** en selecteer **Maak een nieuwe
+   formulierdefinitie**.
+#. Onder de sectie **(Herbruikbare) stapgegevens** vul het volgende in:
+
+   * **Naam**: Stap met herhalende group
+
+#. Scroll naar de sectie **Velden** en klik op **Speciale velden**.
+#. Sleep een **Herhalende group** component op het witte vlak, vul de volgende
+   gegevens in en druk daarna op **Opslaan**:
+
+   * **Basis** > **Label**: Autos
+   * **Weergave** > **Open eerste group als leeg**: *aangevinkt*
+
+#. Klik op **Formuliervelden** en sleep een **Tekstveld** component binnen de herhalende group. Vul de volgende
+   gegevens in en druk daarna op **Opslaan**:
+
+   * **Label**: Kleur
+#. Sleep een **Getal** component binnen de herhalende group. Vul de volgende
```

Comment:
```suggestion
   * **Label**: Kleur

#. Sleep een **Getal** component binnen de herhalende group. Vul de volgende
```
I think it doesn't render properly otherwise